x-pack/test/functional/apps/maps/layer_errors.js

```diff
@@ -90,7 +90,7 @@ export default function ({ getPageObjects }) {
       it('should diplay error message in layer panel', async () => {
```

Comment:
@rashmivkulkarni let's skip just this one test.  Everything else seems to pass in all our test cases.  I've reproduced the same failure on this one test locally by running the current 7.11.0-SNAPSHOT build but don't know how to debug it.

Reply:
skipped `layer_errors` test for the reason above. 
ref: https://github.com/elastic/kibana/issues/69617